packages/fuels-core/Cargo.toml

```diff
@@ -31,6 +31,7 @@ thiserror = { workspace = true, default-features = false }
 uint = { workspace = true, default-features = false }
 
 [dev-dependencies]
+fuel-tx = { workspace = true, features = ["test-helpers", "random"] }
```

Comment:
Can this potentially cause incompatibility issues at some point if someone is interacting with a node instance that doesn't have `test-helpers` enabled?

Reply:
This is a dev-dependency, as such it is only used in our unit tests of `fuels-core`. We need it for `ConsensusParameters::standard()`.